content/applications/essentials/export_import_data.rst

```diff
@@ -2,166 +2,245 @@
+#. When exporting, there is the option to export in two formats: `.csv` and `.xls`. With `.csv`,
+   items are separated by a comma, while `.xls` holds information about all the worksheets in a
+   file, including both content and formatting.
+#. These are the items that can be exported. Use the :guilabel:`> (arrow)` icon to display more
+   sub-field options. Use the :guilabel:`Search` bar to find specific fields more easily. To use the
+   :guilabel:`Search` option more efficiently, click on all the :guilabel:`> (arrows)` to display
+   all fields.
+#. The :guilabel:`+ (plus sign)` icon button is present to add fields to the :guilabel:`Fields to
+   export` list.
+#. The :guilabel:`↕️ (up-down arrow)` to the left of the selected fields can be used to move the
+   fields up and down, to change the order in which they are displayed in the exported file.
+   Drag-and-drop from the :guilabel:`↕️ (up-down arrow)` icon.
+#. #. The :guilabel:`🗑️ (trash can)` icon is used to remove fields. Click on the :guilabel:`🗑️
+   (trash can)` icon to remove the field.
+#. For recurring reports, it is helpful to save export presets. Select all the needed fields, and
+   click on the template drop-down menu. Once there, click on :guilabel:`New template`, and give a
+   unique name to the export just created. The next time the same list needs to be exported, select
+   the related template that was previously saved from the drop-down menu.
 
 .. tip::
-   It’s good to keep the field’s external identifier in mind. For example,
-   *Related Company* is equal to *parent_id*. Doing so helps export
-   only what you would like to import next.
+   It is helpful to know the field's external identifier. For example, :guilabel:`Related Company`
+   in the export user interface is equal to *parent_id* (external identifier). This is helpful
+   because then, the only data exported is what should be modified and re-imported.
 
 .. _import-data:
 
 Import data into Odoo
 =====================
 
+Importing data into Odoo is extremely helpful during implementation, or in times where data needs to
+be updated in bulk. The following documentation covers importing data into an Odoo database.
+
+.. warning::
+   Imports are permanent and **cannot** be undone. However, it is possible to use filters (`created
+   on` or `last modified`) to identify records changed or created by the import.
+
+.. tip::
+   Activating :ref:`developer mode <developer-mode>` changes the visible import settings in the left
+   menu. Doing so reveals an :menuselection:`Advanced` menu. Included in this advanced menu are two
+   options: :guilabel:`Track history during import` and :guilabel:`Allow matching with subfields`.
+
+   .. image:: export_import_data/advanced-import.png
+      :align: center
+      :alt: Advanced import options when developer mode is activated.
+
+   If the model uses openchatter, the :guilabel:`Track history during import` option sets up
```

Comment:
does openchatter need to be formatted in any specific way?

Reply:
This is the format open chatter is show in the tip on Odoo: 
![2024-03-20_16-02](https://github.com/odoo/documentation/assets/91896312/388d25d6-77e7-441a-b174-732a04afaf17)